sources/packages/backend/workflow/test/2021-2022/parttime-assessment/eligibility/parttime-assessment-2021-2022-eligibility-BCAG.e2e-spec.ts

```diff
@@ -103,4 +104,69 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-BCAG
       });
     }
   });
+
+  it("Should determine provincialAwardNetBCAGAmount when awardEligibilityBCAG is true", async () => {
```

Comment:
I see that these E2E tests(created newly) are asserting the final value of the award not just the eligibility.

should it come under *eligibility-BCAG.e2e-spec.ts ? under eligibility/ ?

Reply:
The two tests are not exactly the same. The on in the image determines BCAG eligibility, but line 108 determines provincialAwardNetBCAGAmount.

Reply:
@andrewsignori-aot @guru-aot @andrepestana-aot thoughts?

Reply:
As per the Team conversation let's move these new tests to a new folder (e.g. awards-amounts) and have it added to the 3 current part-time program years folders.

Reply:
Rename the folders' names from "awards-amount" to "awards-amounts" in my latest commit. Thanks guys.